pkg/apis/apps/types.go

```diff
@@ -157,6 +191,12 @@ type StatefulSetSpec struct {
 	// consists of all revisions not represented by a currently applied
 	// StatefulSetSpec version. The default value is 10.
 	RevisionHistoryLimit *int32
+
+	// PersistentVolumeClaimPolicy describes the policy used for PVCs created from
+	// the StatefulSet VolumeClaimTemplates. This requires the
+	// StatefulSetAutoDeletePVC feature gate, which is alpha.
```

Comment:
`This requires the StatefulSetAutoDeletePVC feature gate to be enabled`

Reply:
done